SteamList/Models/FavoriteModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  FavoriteModel.swift
+//  SteamList
+//
+//  Created by Macbook on 05/01/2023.
+//
+
+import Foundation
+
+class FavoriteModel {
```

Comment:
Is there a reason for this model to be a `class` instead of a `struct`?

Reply:
I already deleted this class I am using already defined Game struct instead, but you are right it should've been struct